### PR TITLE
Bug fix

### PR DESCRIPTION
**Pull request by NimaSoroush**
- Bug fixes

### CODE REVIEW

File: src/chromyRunner.test.js [hunk 1]
```diff
@@ -248,31 +252,31 @@ describe('ChromyRunner', () => {
...
+          { name: 'execute', value: () => {} },
         ],
       };
       const result = await run(chromy, globalConfig, newConfig);
       expect(result).toEqual(true);
       expect(chromy.evaluate).toHaveBeenCalledTimes(1);
       expect(loggerCalls[0]).toEqual('waiting for to evaluate function in browser');
     });
-    it('Evaluate: non-function', async () => {
+    it('Ececute: non-function', async () => {
```
Comment (danrr) [inline src/chromyRunner.test.js]: typo

File: src/chromyRunner.test.js [hunk 2]
```diff
@@ -281,4 +285,43 @@ describe('ChromyRunner', () => {
       expect(loggerCalls[0]).toEqual('failed to detect evaluate function');
     });
   });
+  describe('Chromy runner', () => {
+    it('FreezeImge: existing selector', async () => {
```
Comment (danrr) [inline src/chromyRunner.test.js]: typo

File: src/chromyRunner.js [hunk 3]
```diff
@@ -88,7 +88,7 @@ const run = async (chromy, options, test) => {
           return false;
         }
         break;
-      case actions.evaluate:
+      case actions.execute:
         try {
           if (check.function(action.value)) {
             prefixedLogger.log('waiting for to evaluate function in browser');
```
Comment (danrr) [inline src/chromyRunner.js]: the logger here should probably say execute instead of evaluate

Comment (NimaSoroush) [inline src/chromyRunner.js]: yes

File: src/chromyRunner.test.js [hunk 4]
```diff
@@ -26,6 +27,8 @@ jest.mock('./compareImage', () => jest.fn(arg =>
   }),
 ));
 
+jest.mock('./freezeImage', () => jest.fn(() => true));
+
 let loggerCalls = [];
```
Comment (danrr) [inline src/chromyRunner.test.js]: It's probably worth boyscouting this to use mocks for logger and fs

File: src/chromyRunner.test.js [hunk 5]
```diff
@@ -4,6 +4,7 @@ import run from './chromyRunner';
 import logger from './logger';
 import { globalConfig, testConfig, configTypes } from './defaultConfig';
 import actions from './actions';
+import freezeImage from './freezeImage';
 
 jest.mock('chromy', () => jest.fn(() =>
   ({
```
Comment (danrr) [inline src/chromyRunner.test.js]: this mock doesn't need close

File: src/chromyRunner.test.js (hunk 4 above)
Comment (NimaSoroush) [inline src/chromyRunner.test.js]: We have a separate issue for test refactoring. I will do that there

**Pull request merged**